tests/ClientTest.php

```diff
@@ -131,15 +131,20 @@ public function testClient(): void
         $this->assertTrue($foundInList);
 
         // 5. Deactivate
-        $client->deactivate($sandboxId);
-
-        // 6. Get and check if deactivated
-        $response = $client->get($sandboxId);
+        $response = $client->deactivate($sandboxId);
         $this->assertNotEmpty($response);
         $this->assertFalse($response->getActive());
-        $this->assertEmpty($response->getDeletedTimestamp());
```

Comment:
this should still be empty right?

Reply:
Yes, I rather returned the assert back. ;o) (I did not remove it on purpose, it just looks so because of reordering of the API calls. 😄 )